ds_judgements_public_ui/templates/forms/widgets/court_input_option_with_years.html

```diff
@@ -0,0 +1,14 @@
+{% load court_utils %}
+{% if widget.wrap_label %}
+  <div class="structured-search__court-option">
+    <label {% if widget.attrs.id %}for="{{ widget.attrs.id }}"{% endif %}>
+    {% endif %}
```

Comment:
Indentation is misleading.

Reply:
Annoyingly this is being auto-formatted by the pre-commit!

Reply:
See https://github.com/nationalarchives/ds-caselaw-public-ui/pull/1218/files